data/scripts.js

```diff
@@ -79,8 +83,8 @@ function getChannelName(){
               <div class="siimple-grid-col siimple-grid-col--2 siimple-grid-col-sm--12" align=center><div style="width: 100%; padding: 3px; margin-bottom: 2px; margin-top: 2px;" onclick="runShutterCmd('stop', ` + channel_id  + `)" class="siimple-btn siimple-btn--navy">STOP</div></div>
               <div class="siimple-grid-col siimple-grid-col--2 siimple-grid-col-sm--12" align=center><div style="width: 100%; padding: 3px; margin-bottom: 2px; margin-top: 2px;" onclick="runShutterCmd('down', ` + channel_id  + `)" class="siimple-btn siimple-btn--navy">DOWN</div></div>
               <div class="siimple-grid-col siimple-grid-col--2 siimple-grid-col-sm--12" align=center><div style="width: 100%; padding: 3px; margin-bottom: 2px; margin-top: 2px;" onclick="runShutterCmd('shade', ` + channel_id  + `)" class="siimple-btn siimple-btn--navy">SHADE</div></div>
-              <div class="siimple-grid-col siimple-grid-col--2 siimple-grid-col-sm--12" align=center><div style="width: 100%; padding: 3px; margin-bottom: 2px; margin-top: 2px;" onclick="runShutterCmd('set shade', ` + channel_id  + `)" class="siimple-btn siimple-btn--grey">SET SHADE</div></div>
-              <div class="siimple-grid-col siimple-grid-col--2 siimple-grid-col-sm--12" align=center><div style="width: 100%; padding: 3px; margin-bottom: 2px; margin-top: 2px;" onclick="runShutterCmd('learn', ` + channel_id  + `)" class="siimple-btn siimple-btn--grey">LEARN</div></div>
+              <div name="learnBtn" class="siimple-grid-col siimple-grid-col--2 siimple-grid-col-sm--12" align=center><div style="width: 100%; padding: 3px; margin-bottom: 2px; margin-top: 2px;" onclick="runShutterCmd('set shade', ` + channel_id  + `)" class="siimple-btn siimple-btn--grey">SET SHADE</div></div>
+              <div name="learnBtn" class="siimple-grid-col siimple-grid-col--2 siimple-grid-col-sm--12" align=center><div style="width: 100%; padding: 3px; margin-bottom: 2px; margin-top: 2px;" onclick="runShutterCmd('learn', ` + channel_id  + `)" class="siimple-btn siimple-btn--grey">LEARN</div></div>

```

Comment:
changing the attribute from 'display: inline-block' to 'display: inline' from 'label class siimply-label' fixes a display error of the checkbox and label of' save prefix and generate channel serial numbers'. These overlapped on small screens.